manual/python/index.md

```diff
@@ -142,13 +142,25 @@ Moreover, `addressof` can be used in conjunction with `TTree::Branch` from Pytho
 
 - `SetOwnership(obj, python_owns)`: a Python proxy can either own or not own its internal C++ object. If a Python proxy owns its C++ object and the proxy is being destroyed, the C++ object will be deleted too. This ownership can be modified for a given Python proxy with `SetOwnership`. For example, when calling `SetOwnership(obj, False)`, the user makes sure that the C++ object proxied by `obj` will not be deleted by PyROOT when `obj` is garbage collected; this is useful e.g. if the user knows the deletion will happen in C++ and wants to prevent a double delete. This functionality should be used with care not to produce memory leaks.
 
-## Loading user libraries & jitting
+## Loading user libraries & Just-In-Time compilation (JITting)
 
-PyROOT allows to use *any C++ library* from Python, not only ROOT. This is possible thanks to the automatic and dynamic bindings between Python and C++ that PyROOT provides: without any prior generation of wrappers, at execution time, PyROOT can load and call into C++ code. The following sub-sections cover different supported scenarios for dynamic C++ code invocation.
+PyROOT allows to use *any C++ library* from Python, not only ROOT's libraries. This is possible thanks to the automatic and dynamic bindings between Python and C++ that PyROOT provides.
+Without any prior generation of wrappers, at execution time, PyROOT can load C++ code, and call its functions.
 
-### Jitting strings with code
+This allows for writing high-performance C++, compiling it, and using it from Python.
+The following options are available, ordered by complexity and performance:
 
-ROOT comes with a [C++ interpreter]({{ '/manual/first_steps_with_root/#using-the-interactive-c-interpreter-cling' | relative_url }}) to which we can request to process a given piece of C++ code. Sometimes, if such code is short (e.g. the definition of a small function or class) or for rapid exploration/debugging, it is enough the place the C++ code in a Python string and give it to the interpreter. The code will be just-in-time compiled (jitted) and immediately available for invocation, as shown in the example below, which calls the constructor of the C++ class `A` and function `f` from Python after defining them via the interpreter.
+1. [Just-in-time compilation of small strings](#JITString): Small functions and classes to be used from Python, testing
```

Comment:
instead of just `, testing` perhaps `. Useful for testing/rapid exploration/debugging` ?

Reply:
```suggestion
1. [Just-in-time compilation of small strings](#JITString): Small functions and classes to be used from Python. Also useful for testing and debugging.
```